core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -467,6 +471,22 @@ public <T> ServerBuilder childChannelOption(ChannelOption<T> option, T value) {
         return this;
     }
 
+    /**
+     * The {@link Consumer} that customizes the Netty {@link ChannelPipeline}.
+     * This customizer is no-op by default.
```

Comment:
Could you note that:
`this customizer is run right before HTTP(S) codecs are configured.`